### PR TITLE
Adding Tab Location "left"

### PR DESCRIPTION
**Pull request by TakuroOnoda**
terminus_sidetab

#641 #825

Hello.
I really wanted a side-tab so I implemented it.
I don't understand the whole structure, so I'd like to offer this as an idea.
If possible, I would like the contributors to refine this PR. How about?
Also, If the implementation of left tab is already underway somewhere, I'll wait quietly.
Thank you :smile:

### CODE REVIEW

File: terminus-core/src/components/appRoot.component.pug [hunk 1]
```diff
@@ -3,99 +3,133 @@ title-bar(
...
-            dnd-sortable-container,
-            [sortableData]='app.tabs',
-        )
-            tab-header(
-                *ngFor='let tab of app.tabs; let idx = index',
-                dnd-sortable,
-                [sortableIndex]='idx',
-                (onDragStart)='onTabDragStart()',
-                (onDragEnd)='onTabDragEnd()',
+.wrap
```
Comment (TakuroOnoda) [inline terminus-core/src/components/appRoot.component.pug:6]: Add one div to make the whole structure a flex column

File: terminus-core/src/components/appRoot.component.pug [hunk 2]
```diff
@@ -3,99 +3,133 @@ title-bar(
...
+  .tab-bar.vertical(
```
Comment (TakuroOnoda) [inline terminus-core/src/components/appRoot.component.pug:7]: I wanted to make tab-bar one component, but it was difficult and I couldn't do it well :confounded:

File: terminus-ssh/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "webpack --progress --color",
     "watch": "webpack --progress --color --watch",
-    "postinstall": "xcopy /i node_modules\\ssh2\\util\\pagent.exe util\\"
+    "postinstall": "xcopy /i /y node_modules\\ssh2\\util\\pagent.exe util\\"
```
Comment (TakuroOnoda) [inline terminus-ssh/package.json:13]: Fixed because when I executed a command `node scripts\build-native.js` in windows, it stopped responding to xcopy

**Comment by Eugeny** [comment]
Very nice! I've refined the PR a bit and added support for right-side tabs as well

**Comment by Eugeny** [comment]
@allcontributors please add @TakuroOnoda for code

**Pull request merged**